apps/aecore/lib/aecore/governance/governance_constants.ex

```diff
@@ -42,6 +44,31 @@ defmodule Aecore.Governance.GovernanceConstants do
 
   @max_txs_per_block 10_946
 
+  @default_tx_gas_price 15_000
+
+  @allowed_txs_types [
```

Comment:
if any new transaction type has the need to be added here, you should add it to the dev documentation